contents/projectManagement/AgileDevelopment.md

```diff
@@ -0,0 +1,127 @@
+>1. Exit first airplane via arrival gate.
+>2. Determine departure gate.
+>3. Walk to the departure gate.
+>4. Board second airplane via departure gate.
+>In a small airport this prestored plan would work just fine. But in a larger airport, such as >Kennedy Airport, if the arrival and departure gates are in different terminals, the plan would >have to be refitted (e.g., the planner might have to take a shuttle between terminals). The >problem of adaptive planning (reasoning) is to take a prestored plan (piece of knowledge) such
+>as the one shown above, and apply it to a novel set of circumstances...
+>
+>-- from [Adaptive Planning](http://onlinelibrary.wiley.com/store/10.1207/s15516709cog1203_3/asset/s15516709cog1203_3.pdf?v=1&t=jeef0qzp&s=ce2d7fe621eee0a3d7a4a25867e84fb22668d304) by Richard Alterman
+
+---
+
+Secondly, agile development is "iterative" because it breaks product development work down into short cycles known as [iterations](https://www.agilealliance.org/glossary/iteration/). Each iteration involves going through planning, designing, coding and testing to result in a working product at the end of an iteration. However, a single iteration may not have added enough functionality to warrant a new release, so in reality multiple iterations are often required to release a new product or feature. Here is a diagram to illustrate this concept:
+
+![Agile Development - Iterations](images/projectManagement-agileDev-iterations.jpg)
+
+---
+
+Thirdly, agile development is "people-centric" because it focuses heavily on collaboration and effective communication. In fact, there are many recommendations on how to form a good agile team, but the general consensus is that agile development involves [several crucial team roles](https://apiumhub.com/tech-blog-barcelona/building-agile-team/) that work in close relation with each other.
+
+In particular, agile development also zooms in on direct and simple conversations as the main mode of communication. For example, popular agile frameworks like [Scrum](https://www.scrum.org/resources/what-is-scrum?gclid=EAIaIQobChMIrJLEjuPV2QIV1BuPCh17KwH_EAAYASAAEgKXPfD_BwE) and [Kanban](https://www.atlassian.com/agile/kanban) typically have daily meetings called stand-ups where participants meet for no more than 15 minutes while standing up. Each team member simply answers three questions: "what did I complete yesterday", "what will I work on today", and "am I blocked by anything?", quickly informing everyone of what is going on across the team.
+
+---
+
+The key characteristics then enable our software projects to gain numerous benefits when agile development is adopted to manage the project.
+
+## Why adopt agile development?
+
+There are quite a number of reasons why you might want to adopt development, but here are three of the most pertinent reasons why:
+
+1. **Agile development can increase user satisfaction with the product.**
+
```

Comment:
Might be good to have interesting examples here by real companies, if any.

Reply:
I have decided to create a new subsection with a full example at the conclusion, so hopefully it will serve the purpose here as well :)